README.md

```diff
@@ -1 +1,55 @@
-# potential-eureka
\ No newline at end of file
+# module one project 
+this is a project to learn the basics of how to use linters and workflows
+
+
+## HTML AND CSS PRACTICE SESSION
+
+In this practice session we:
+
+### added HTML and CSS file to Github
+### created a linter to check for error in the HTML and CSS codes
```

Comment:
[Optional]

- Don't use bold text much. It makes unreadable after a certain limit.